src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -178,6 +189,40 @@ internal DecompressionMethods AutomaticDecompression
             }
         }
 
+        internal CookieUsePolicy CookieUsePolicy
```

Comment:
Why not expose the same UseCookies that's on HttpClientHandler?  I've been viewing that type as a simple facade/passthrough that wraps this, simply delegating with minimal ceremony to the exact matching properties on CurlHandler.